resources/styles/book-content/mixins.less

```diff
@@ -33,3 +33,10 @@
   // to fit beside it, otherwise it will overlay on top of the next element
   min-height: 8rem;
 }
+
+.book-content-step-size(@vertical: @tutor-card-body-padding-vertical) {
+  padding: @vertical @tutor-card-body-padding-horizontal;
+  @media (max-width: @tutor-card-padding-collapse-breakpoint) {
```

Comment:
Should this step sizing be done somehow similarly? @darrenmason is doing something similar for https://github.com/openstax/tutor-js/pull/298#discussion_r31087563